lib/jsdom/living/post-message.js

```diff
@@ -21,11 +21,11 @@ module.exports = function (message, targetOrigin) {
   }
 
   // TODO: event.source - requires reference to source window
-  // TODO: event.origin - requires reference to source window
   // TODO: event.ports
   // TODO: event.data - structured clone message - requires cloning DOM nodes
   const event = new this.MessageEvent("message", {
-    data: message
+    data: message,
+    origin: targetOrigin
```

Comment:
Please read the spec. The origin property of the event is completely unrelated to targetOrigin. IT's the source origin. The whole point of postMessage is cross-origin messaging.

If you want to introduce origin, you're going to need to start importing and passing more web platform tests. I think it'll be easier not to make this unrelated change, introducing a new feature, here in this PR, which is about fixing a typo.